tools/nodejs/lib/duckdb.d.ts

```diff
@@ -171,6 +171,45 @@ export class Database {
   ): Promise<void>;
 }
 
+type TypeInfo = {
+  id: string,
+  sql_type: string,
+  alias?: string,
+} | {

```

Comment:
These should probably be made to be standalone exported types/interfaces that are then used in TypeInfo